BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -48,9 +48,8 @@ private static BraintreeClientParams createDefaultParams(Context context, String
      *
      * @param context       Android Context
      * @param authorization The tokenization key or client token to use.
-     * @throws InvalidArgumentException If the tokenization key or client token is not valid, or cannot be parsed.
```

Comment:
I think it is still useful for this docstring to call out how an error will surface if an invalid credential is provided.

Something like:

```
     * @param authorization The tokenization key or client token to use. If an invalid key is provided, a `BraintreeException` will be returned in the `HttpResponseCallback`.
```